src/PhpDoc/StubValidator.php

```diff
@@ -82,7 +82,7 @@ public function validate(): array
 		$errors = [];
 		foreach ($this->stubFiles as $stubFile) {
 			$tmpErrors = $fileAnalyser->analyseFile(
-				$stubFile,
+				str_replace(DIRECTORY_SEPARATOR, '/', $stubFile),
```

Comment:
I have a doubt on this one. While the PHPStan\Levels\StubValidatorIntegrationTest Test pass multiple times in a row, I think another test is deleting the "stubValidator-0.json" file.

If I'm not mistaken, the tests pass on the first try, but on the second, it complains about the file missing. However, I couldn't fix the issue without that

Reply:
The tests extending LevelsTestCase are self-updating - if the expected contents of JSON files do not match, they are updated and the test fails. When running the 2nd time on the updated JSON files, the test will pass. So this change is breaking something if the JSON files are getting updated.

Reply:
(I don't think this change should be here, only test asserts should be updated.)

Reply:
Yeah, I noticed that, when the test failed on first try, they passed on the second. What's weird this time is that the test pass on first try but fail after. I will investigate more on this

Reply:
> (I don't think this change should be here, only test asserts should be updated.)

You're right, only the tests should change. I'll revert this line.

But to fix the actual issue, I'm kinda stuck. On LevelsTestCase::testLevels we use exec() to launch phpstan. This will always return Windows style path on Windows. And later, we compare this to a file generated on linux.

So we have to translate a path from one style to another somehow. Unfortunately, the FileHelper class use the DIRECTORY_SEPARATOR and will always normalize path on the same style.

This means we have to create a function to normalize a path with the opposite style. (To summarize, transform a Windows path to a Linux path) and then convert all the path in the phpstan output. Do you agree with that?